environment-dev-win.yml

```diff
@@ -0,0 +1,24 @@
+name: foyer-dev
+channels:
+  - conda-forge
+  - omnia
+dependencies:
+  - bump2version
+  - codecov
+  - lark-parser
+  - lxml
+  - mbuild
+  - networkx>=2.0
+  - openmm>=7.5
+  - parmed
+  - pip
+  - pytest
+  - pytest-azurepipelines
+  - pytest-cov
+  - pytest-timeout
+  - python<=3.8
+  - requests
+  - requests-mock
+  - pip:
+    - "--editable=git+https://github.com/rsdefever/GAFF-foyer.git#egg=gafffoyer"
```

Comment:
Once we resolve the issues we found pertaining to #411, we can merge, but im pretty sure we at least found out that we dont need both `antefoyer` and `GAFF-foyer` installed at the same time since they expose the same name for an entry_point.